### DISCUSSION TITLE
Version high-low part of SQL SERVER

### DISCUSSION THREAD

**Discussion by acappelletti** [trimble-oss/dba-dash#786] [General]
I don't understood this two columns?
![image](https://github.com/trimble-oss/dba-dash/assets/43847440/3fa8765a-2ffa-4ca5-8f9b-1ed80dd70f87)

Alen

**Comment by DavidWiseman** [comment]
The values come from sys.configurations.  They are not very well [documented](https://github.com/MicrosoftDocs/sql-docs/issues/5347).  It does seem that they are for internal use only and the values can vary between SQL versions.  DBA Dash incorrectly highlights this as a configured value as the default values have changed.  These settings should probably be ignored.

**Comment by DavidWiseman** [comment]
#790 created for updating DBA Dash

**Reply by acappelletti** [reply]
OK Thanks David

**Comment by DavidWiseman** [comment]
Included in 2.52.0 🚀

**Reply by acappelletti** [reply]
Great David... I'm going to download it right now!
